scripts/lint/lint-markdown.js

```diff
@@ -284,6 +284,8 @@ const opts = {
         MD028: false,
         // Allow indentation in unordered lists.
         MD007: false,
+        // Allow bare URLs.
+        MD034: false,
```

Comment:
This was necessary to make some `figure` shortcodes work. Seems all right, but let me know if this is a particularly bad thing to do. (And if so, how to escape it on a case-by-case basis, as I couldn't seem to figure this out.)